### PR TITLE
Chore/web auth prototype feedback

### PR DESCRIPTION
**Pull request by rolznz**

### CODE REVIEW

File: src/extension/background-script/connectors/alby.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default class Alby implements Connector {
   }
 
   async init() {
-    return this.authorize();
```
Comment (pavanjoshi914) [inline src/extension/background-script/connectors/alby.ts]: we have to return here. then access token and refresh token will be returned to ```validateaccount.js``` and then further stored in the connector config

next routing will be done to ```selectAccount```` where authorize function will be again executed. as we dont have any stored tokens oauth flow will launch again and hence for any action oauth flow will launch again

**Review by pavanjoshi914 — CHANGES_REQUESTED** [review]

File: src/extension/background-script/connectors/alby.ts (hunk 1 above)
Comment (rolznz) [reply src/extension/background-script/connectors/alby.ts]: This is not how the connectors work normally. The init function should follow the connector interface which returns Promise<void>. Unless we have a good reason we should probably not modify the connector interface.

Which IDE do you use? do you see errors here?

Comment (pavanjoshi914) [reply src/extension/background-script/connectors/alby.ts]: The connector is initiated the first time in the validateAccount and then stored in config.
 Other connectors don't need tokens to be stored but the Alby connector needs  this is what we can do in the most simple way 

 I don't think :thinking: the question of consistency arises here from the above assertion. Also if we go another way we might increase lines of code.
I just simply returned the values and added it to the config when the connector initializes.

other connectors does the same eg. lndhub.ts
image

Comment (rolznz) [reply src/extension/background-script/connectors/alby.ts]: For now I have undone it. Maybe we can update the init() return value in the connector interface

**Pull request merged**